### ISSUE TITLE
Error when uploading a Transactional Emails logo

### ISSUE THREAD

**Issue by chaplynsky** [magento/magento2#20091]
### Preconditions (*)

1. Magento 2.3
2. PHP 7.1

### Steps to reproduce (*)

1. Go to Content>Configuration>edit theme>Transactional Emails
2. Upload logo

### Expected result (*)

1. Logo gets uploaded and is visible

### Actual result (*)

Logo is NOT uploaded and the following error is displayed:
_"A technical problem with the server created an error. Try again to continue what you were doing. If the problem persists, try again later."_

**Comment by magento-engcom-team** [comment]
Hi @chaplynsky. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@chaplynsky do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by chaplynsky** [comment]
@magento-engcom-team give me 2.3 instance

**Comment by chaplynsky** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @chaplynsky. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by shikhamis11** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by cotiga** [comment]
A technical problem with the server created an error. Try again to continue what you were doing. If the problem persists, try again later.
IN CONSOL
JQMIGRATE: deferred.pipe() is deprecated

#19681 #19474 #19883 #19687 #19615 #19474...

Always the same problem

I have an new clean install with
Magento 2.3 (composer instal)
ON
Ubuntu 18.04
PHP 7.2
MySQL 5.7
Nginx

I think it's time to really focus on this problem, thanks

**Comment by chaplynsky** [comment]
@cotiga yeah, this seems to be an issue that keeps coming back anywhere a logo is being uploaded. You can take a look at my [fix ](https://github.com/chaplynsky/magento2/commit/ba325135318e2c856e5300d7139f529662679d3e) for the Transactional Emails logo upload. After you apply and  run upgrade, compile, static deploy, reindex, cache clean - you'll be able to upload the logo.

And [here is the fix for the header logo upload ](https://github.com/magento/magento2/pull/18851).

**Comment by magento-engcom-team** [comment]
@shikhamis11 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97462` were created

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by magento-engcom-team** [comment]
Hi @chaplynsky. Thank you for your report. 
The issue has been fixed in magento/magento2#20092 by @chaplynsky in 2.3-develop branch
Related commit(s):
 - [ba325135318e2c856e5300d7139f529662679d3e](https://github.com/magento/magento2/commit/ba325135318e2c856e5300d7139f529662679d3e)

The fix will be available with the upcoming  release.
